proto/zitadel/action/v2beta/action_service.proto

```diff
@@ -484,67 +552,57 @@ message GetTargetRequest {
 }
 
 message GetTargetResponse {
-  GetTarget target = 1;
+  Target target = 1;
 }
 
-message SearchTargetsRequest {
-  optional zitadel.object.v3alpha.Instance instance = 1 [
-    (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_field) = {
-      default: "\"domain from HOST or :authority header\""
-    }
-  ];
-  // list limitations and ordering.
-  optional zitadel.resources.object.v3alpha.SearchQuery query = 2;
+message ListTargetsRequest {
+  // List limitations and ordering.
+  optional zitadel.pagination.v2beta.ListQuery query = 1;
   // The field the result is sorted by. The default is the creation date. Beware that if you change this, your result pagination might be inconsistent.
-  optional TargetFieldName sorting_column = 3 [
+  optional TargetFieldName sorting_column = 2 [
     (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_field) = {
       default: "\"TARGET_FIELD_NAME_CREATION_DATE\""
     }
   ];
   // Define the criteria to query for.
-  repeated TargetSearchFilter filters = 4;
+  repeated TargetSearchFilter filters = 3;
 }
 
-message SearchTargetsResponse {
-  zitadel.resources.object.v3alpha.ListDetails details = 1;
-  repeated GetTarget result = 2;
+message ListTargetsResponse {
+  zitadel.pagination.v2beta.ListDetails details = 1;
+  repeated Target result = 2;
 }
 
 message SetExecutionRequest {
-  optional zitadel.object.v3alpha.Instance instance = 1 [
-    (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_field) = {
-      default: "\"domain from HOST or :authority header\""
-    }
-  ];
-  Condition condition = 2;
-  Execution execution = 3;
+  Condition condition = 1;
+  Execution execution = 2;
```

Comment:
please add some descriptions
also is it intended that i can specify a condition and an execution, which again has a condition?
![image](https://github.com/user-attachments/assets/f4e8b33e-6db8-4b78-901f-aef2faa4abaa)

Reply:
Wrong import, corrected the request message.

Reply:
ok looks better now
still for my understanding: is it intended that i can include for example a function or event condition in request or response execution and vice versa?